### DISCUSSION TITLE
How to remove y axis white space in horizontal bar chart

### DISCUSSION THREAD

**Discussion by mjanulaitis** [Live-Charts/LiveCharts2#920] [General]
I have created a horizontal bar chart based off of the race sample, however there is white space above and below the first and last items.  How can I remove the white space?

![image](https://user-images.githubusercontent.com/11945074/216334022-6b4631ed-8c4d-45e0-8ced-75216f8f7f6d.png)

**Comment by mjanulaitis** [comment]
Status?

**Comment by aaronshore** [comment]
You can set the **Height** to get rid of the white space. I have 2 rows so I used Height = 200 to get rid of the white space.
